.vscode/schemas/meta.schema.json

```diff
@@ -70,6 +70,15 @@
 						"TAnyField"
 					]
 				}
+			},
+			"manual": {
```

Comment:
Not sure if `manual` is the most future-proof name for this, we might well want to link to other pages too (cookbook, API docs...).